bin/fin

```diff
@@ -520,9 +561,19 @@ docker-compose ()
 		[[ ! -O "$SSH_AUTH_SOCK_DIR" ]] && sudo chown $(id -u):$(id -g) "$SSH_AUTH_SOCK_DIR"
 	fi
 
-	# Call regular docker-compose.
-	# "command <binary>" calls the binary directly ignoring any overriding shell functions
-	command docker-compose "$@"
+	if is_compose_v2; then
+		# Call docker compose (v2)
+		# As a plugin for docker cli, it has to be called as "/path/to/binary compose"
+		# This would not be necessary if the binary was installed in the default location
+		# (~/.docker/cli-plugins/docker-compose). However, we install it in ~/.docksal/bin/docker-copose-v2
+		# to prevent conflicts with system binaries.
+		# "command <binary>" calls the binary directly ignoring any overriding shell functions
+		command docker-compose compose "$@"
+	else
+		# Call regular docker-compose.
+		# "command <binary>" calls the binary directly ignoring any overriding shell functions
+		command docker-compose "$@"
```

Comment:
I guess again do we need this if we are forcing them to use v2?

Reply:
Removed all references and support for Docker Compose v1